### PR TITLE
[12.0][MIG] crm_operating_unit

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of crm_operating_unit to v12

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
I found a problem (not sure if it is due sales team module or this one):

Created 2 teams, one for MainOU and one for b2b
User admin is added only to MainOU
User admin is able to create leads for teamb2b and OU b2b

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review on local

File: crm_operating_unit/models/crm_lead.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class CRMLead(models.Model):
+
+    _inherit = 'crm.lead'
+
+    @api.model
+    def _get_default_operating_unit(self):
+        team = self.env['crm.team']._get_default_team_id()
+        if team.operating_unit_id:
+            return team.operating_unit_id
+        else:
```
Comment (nikul-serpentcs) [inline crm_operating_unit/models/crm_lead.py]: No need to `else`
```
if team.operating_unit_id:
    return team.operating_unit_id
return self.env['res.users'].operating_unit_default_get(self._uid)
```

File: crm_operating_unit/views/crm_lead_view.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <data>
```
Comment (nikul-serpentcs) [inline crm_operating_unit/views/crm_lead_view.xml]: @ageficent no need

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

File: crm_operating_unit/views/crm_lead_view.xml (hunk 2 above)
Comment (AdriaGForgeFlow) [reply crm_operating_unit/views/crm_lead_view.xml]: Done :+1:

**Review by CasVissers-360ERP — APPROVED** [review]
Functionally tested. Works great.

**Review by Nikul-Chaudhary — APPROVED** [review]
Code Review LGTM

**Comment by AaronHForgeFlow** [comment]
/ocabot merge

Comments attended

**Pull request merged**